posthog/client.py

```diff
@@ -127,25 +129,36 @@ def identify(self, distinct_id=None, properties=None, context=None, timestamp=No
             "uuid": uuid,
         }
 
-        return self._enqueue(msg)
+        return self._enqueue(msg, geoip_disable)
 
-    def get_feature_variants(self, distinct_id, groups=None, person_properties=None, group_properties=None):
-        resp_data = self.get_decide(distinct_id, groups, person_properties, group_properties)
+    def get_feature_variants(
+        self, distinct_id, groups=None, person_properties=None, group_properties=None, geoip_disable=None
+    ):
+        resp_data = self.get_decide(distinct_id, groups, person_properties, group_properties, geoip_disable)
         return resp_data["featureFlags"]
 
-    def _get_active_feature_variants(self, distinct_id, groups=None, person_properties=None, group_properties=None):
-        feature_variants = self.get_feature_variants(distinct_id, groups, person_properties, group_properties)
+    def _get_active_feature_variants(
```

Comment:
Don't think this works for the case when someone calls `posthog.capture(send_feature_flags=True, geoip_disable=False)`, or is tested yet?

Reply:
huh? what exactly would that test? If we're checking to see if setting a method `geoip_disable` param overrides the init one then the test in `test_feature_flags` should cover that, right?

Reply:
What is the problem/issue here of why `posthog.capture(send_feature_flags=True, geoip_disable=False)` won't work?

Reply:
https://github.com/PostHog/posthog-python/blob/geoip-disable/posthog/client.py#L211

This is separate from `test_feature_flags`, it's for `test_capture`. In the line above^, we're not passing in `geoip_disable` coming from capture, which is why anything the user passes in there is discarded when it comes to `_get_active_feature_variants`

Reply:
oh I thought that would've been fine since it's a capture call that still hits the logic in enqueue for it

Reply:
but I guess what you're saying here is the `feature flags` that returns will be incorrect because `geoip_disable` isn't passed there, yes?

Reply:
correct, so it will rightly not add geoip properties to the captured event, but wrongly add feature flags.

Reply:
Can we lead with that next time? 😁 I'm better at working backwards from the initial problem than building up to the problem, if that makes sense?

Reply:
Not quite sure I get it. Could you give me another example of what this means?